spec/support/corpus.rb

```diff
@@ -58,18 +66,16 @@ def verify_mutation_coverage
       #
       # @raise [Exception]
       #   otherwise
-      #
-      # rubocop:disable AbcSize
       def verify_mutation_generation
         checkout
         start = Time.now
-        paths = Pathname.glob(repo_path.join('**/*.rb')).sort_by(&:size).reverse
+
         options = {
```

Comment:
I think for the same reason not an instance method as https://github.com/mbj/mutant/pull/508#discussion_r48656994, not frozen because the last time I tried `parallel` did mutate the input, I'm happy to retry freezing in another iteration, but as I want to use mutants parallel interface in the future likely this code will go away before.